### ISSUE TITLE
Node 6 support destructuring and function name

### ISSUE THREAD

**Issue by Pana** [jhen0409/babel-preset-es2015-node6#4]
FYI

**Comment by jhen0409** [comment]
Thanks!

The `destructuring` is related to #2, I believe it will be removed in the near future.

Currently the `function-name` has many of the features we need to enable `--harmony-function-name` in v8 5.0, so I still keep it.

**Comment by olalonde** [comment]
I'm not sure I understand why `function-name` is included. It seems to be supported natively in Node v6.

**Comment by jhen0409** [comment]
@olalonde you can see `function "name" property` in [ES6 compatibility table](https://kangax.github.io/compat-table/es6/#node6), currently babel has two features we need enable `--harmony-function-name` on node 6:
- variables (function)
- object methods (function)

**Comment by olalonde** [comment]
Ah ok, thanks.

**Comment by christophehurpeau** [comment]
Yes, but as you can see [in node.green](http://node.green/#function--name--property) the function name features are supported in node 6.5 because [v8 was updated to 5.1](https://github.com/nodejs/node/pull/8054)

**Comment by christophehurpeau** [comment]
Maybe we could use the preset `es2015-node6/node6.5` ?

**Comment by jhen0409** [comment]
@christophehurpeau, sorry for delay..

I think with the latest version will be better, so may I'll provide like `less-than-6.5` preset.
